modules/installation-osp-kuryr-octavia-configuration.adoc

```diff
@@ -240,3 +240,99 @@ If Kuryr uses `ovn` instead of Amphora, it offers the following benefits:
 * Reduced network latency.
 * Increased service creation speed by using OpenFlow rules instead of a VM for each Service.
 * Distributed load balancing actions across all nodes instead of centralized on Amphora VMs.
+
+
+[id="installation-osp-kuryr-octavia-driver_upgrade_{context}"]
+== Upgrading Octavia provider drivers after installation
+
+You can upgrade a cluster that is deployed on {rh-openstack-first} from the Octavia Amphora
+provider driver to the Octavia OVN provider driver.
+
+[IMPORTANT]
+====
+Kuryr replaces existing load balancers after you change provider drivers. This process
+results in some downtime.
+====
+
+.Prerequisites
+
+* Install the {rh-openstack} CLI, `openstack`.
+
+* Install the {product-title} CLI, `oc`.
+
+* Verify that the Octavia OVN driver on {rh-openstack} is enabled.
++
+[TIP]
+====
+To view a list of available Octavia drivers, on a command line, enter `openstack loadbalancer provider list`.
+You should see the `ovn` driver in the command's output.
+====
+
+.Procedure
+
+To change from the Octavia Amphora provider driver to Octavia OVN:
+
+. Open the `kuryr-config` ConfigMap. On a command line, enter:
++
+----
+$ oc -n openshift-kuryr edit cm kuryr-config
+----
+
+. In the ConfigMap, delete the line that contains `kuryr-octavia-provider: default`. For example:
++
+----
+...
+kind: ConfigMap
+metadata:
+  annotations:
+    networkoperator.openshift.io/kuryr-octavia-provider: default <1>
+...
+----
+<1> Delete this line. The cluster will regenerate it with `ovn` as the value.
++
+After making this modification, wait for the `kuryr-controller` and `kuryr-cni` pods to redeploy. This process may take several minutes.
```

Comment:
this should be "wait until the Cluster Network Operator detects the modification and redeploy the kuryr-controller and kuryr-cnis"